quickscreen/summary/summary_classes.py

```diff
@@ -163,6 +177,12 @@ def __init__(self, df, type="columns"):
         >>> Stats(pd.DataFrame(df), "rows")
         """
         Df_Info.__init__(self, df, type)
+        try:
```

Comment:
This seems funky

Reply:
Basically, the parent class initializing creates an empty class value with no attributes as it fails gracefully avoiding a crash. I think it makes more sense to have the class fail not gracefully but this is what I came up with so that if that parent class doesn't initialize properly, this one won't either and will throw an exception and also fail gracefully.